_includes/why.html

```diff
@@ -3,20 +3,14 @@
 
   <h2 id="why" class="section-header">Why</h2>
 
-  <p>
-    We would like to help improve diversity and the gender ratio in the Cape Town tech community.
-  </p>
+  <p>We want to push for all kinds of diversity in the Cape Town tech community: gender, race, sexual orientation, ability, and class. Women were the first population that RailsBridge focused on, but aren't the last.</p>
 
-  <p>
-    We want to show how programming skills can be useful, empowering, and fun, particularly when applied to the web.
-  </p>
+  <p>We believe that the people making technology should accurately reflect the diversity of those using it.</p>
 
-  <p>
-    We want to bring more people into the local Ruby community, and encourage more user groups to think about diversity and outreach.
-  </p>
+  <p>We want to show how programming skills can be useful, empowering, and fun, particularly when applied to the web.</p>
```

Comment:
Maybe too philosophical, but consider mentioning that the reason diversity is important in web programming, is that web programming is a social problem solving endeavour; broader perspective and understanding lead to better solutions.